frontend/.eslintrc.cjs

```diff
@@ -2,17 +2,18 @@ module.exports = {
   root: true,
   env: { browser: true, es2020: true },
   extends: [
-    'eslint:recommended',
-    'plugin:@typescript-eslint/recommended',
-    'plugin:react-hooks/recommended',
+    "eslint:recommended",
```

Comment:
I prefer single-quotes. Change prettier so that we use single-quotes.

Reply:
Actually, I like single-quotes for javascript, but not for CSS or HTML. I recommend you follow this: https://stackoverflow.com/questions/55430906/prettier-single-quote-for-javascript-and-json-double-quote-for-html-sass-and-c